### PR TITLE
Feature/vcm 12 Create a React-Native Compatible VC-JS

### PR DESCRIPTION
**Pull request by GabrielKidane96**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
-# Verifiable Credentials JS Library _(vc-js)_
-
-[![Build Status](https://travis-ci.org/digitalbazaar/vc-js.png?branch=master)](https://travis-ci.org/digitalbazaar/vc-js)
-[![NPM Version](https://img.shields.io/npm/v/vc-js.svg?style=flat-square)](https://npm.im/vc-js)
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>
+  Verifiable Credentials JS Library (react-native-vc-js)
```
Comment (srmalley-sphereon) [inline README.md]: I think we can say something about React-Native in here. We should make the README title in line with what the other libraries say. For instace, `react-native-credentials-context` says: A React Native supported W3C Credentials Context Library

File: package.json [hunk 2]
```diff
@@ -112,5 +112,15 @@
     "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm run test-node",
     "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=text-lcov npm run test-node > coverage.lcov",
     "coverage-report": "nyc report"
-  }
+  },
+  "bundledDependencies": [
```
Comment (srmalley-sphereon) [inline package.json:116]: What does this do? In addition, I remember on `jsonld-signatures` we needed a `resolutions` section in the `package.json` to enforce resolutions of sub-dependencies. Do we not need that here?

**Review by srmalley-sphereon — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (GabrielKidane96) [reply README.md]: done

File: package.json (hunk 2 above)
Comment (GabrielKidane96) [reply package.json:116]: This makes sure that the dependencies get packaged when running `npm pack`. No, we do not need a resolution.

**Review by srmalley-sphereon — APPROVED** [review]

**Pull request merged**